stories/3-Button.stories.tsx

```diff
@@ -26,6 +27,19 @@ storiesOf("Button", module).add("Button States", () => (
       {"Click Me - shouldDisplaySpinner"}
     </Button>
 
+    <br />
+
+    <StateProvider initialState={false}>
+      {(state, setState) => (
+        <Button
+          type={"button"}
+          onClick={() => setState(true)}
+          shouldDisplaySpinner={state}>
+          {"Click Me - shouldDisplaySpinner"}
+        </Button>
+      )}
+    </StateProvider>
```

Comment:
Do we need this one? We already have an example for `shouldDisplaySpinner` above